packages/modelence/src/auth/user.ts

```diff
@@ -59,6 +59,26 @@ export default new Module('_system.user', {
     type: 'ip',
     window: time.days(1),
     limit: 200,
+  }, {
+    bucket: 'signin',
+    type: 'ip',
+    window: time.minutes(15),
+    limit: 20,
+  }, {
+    bucket: 'signin',
+    type: 'ip',
+    window: time.days(1),
+    limit: 200,
+  }, {
+    bucket: 'verification',
```

Comment:
Shouldn't the bucket be the user for the verification, since we have the user at that point already?